pupil-spa/src/app/services/device/device.service.spec.ts

```diff
@@ -37,9 +41,20 @@ describe('DeviceService', () => {
   });
 
   it('captures device info to localStorage',  async () => {
+    // test setup
+    service.deleteDeviceCookie();
+    service.setupDeviceCookie();
+
+    // exec
     await service.capture();
+
+    // tests
     const deviceInfo = storageService.getDeviceData();
     expect(deviceInfo).toBeTruthy();
+    expect(deviceInfo.deviceId.length).toBe(36); // uuid as string
```

Comment:
new linter will fail this in favor of `toHaveLength(36)`

Reply:
That is not a valid `jasmine` matcher for version 2.8.
https://jasmine.github.io/api/2.8/matchers.html

Are you thinking of `jest`? 😄

Reply:
ahh jeez, yes i'm auto thinking you are in `tslib` 🤦‍♂️ 😄